data/json/mapgen/moonshine_camp.json

```diff
@@ -1,84 +1,77 @@
 [
+  {
+    "type": "palette",
+    "id": "moonshine_palette",
+    "terrain": {
+      "d": "t_dirtfloor",
+      "^": "t_dirtfloor",
+      "H": "t_dirtfloor",
+      "+": "t_dirtfloor",
+      "O": "t_dirtfloor",
+      "a": "t_region_groundcover_barren",
+      "c": "t_region_groundcover_barren",
+      "_": "t_region_groundcover_barren",
+      "|": "t_wall_wood",
+      "t": "t_dirtfloor",
+      "b": "t_dirtfloor",
+      "w": "t_region_groundcover_barren"
+    },
+    "furniture": {
+      "+": [ [ "f_canvas_door", 3 ], "f_canvas_door_o" ],
+      "H": "f_wood_keg",
+      "O": "f_fvat_empty",
+      "^": "f_still",
+      "a": [ "f_ash", "f_firering" ],
+      "c": "f_camp_chair",
+      "b": "f_camp_chair",
+      "w": "f_wood_keg",
+      "t": "f_tourist_table"
+    },
+    "items": {
+      " ": [ { "item": "trash", "chance": 1 }, { "item": "clutter_yard", "chance": 1 } ],
+      "t": { "item": "moonshine_brew", "chance": 50 }
```

Comment:
Probably should increase this chance to 70 as the existing items are 65 and 75

Reply:
Makes sense.